integration-test.sh

```diff
@@ -85,9 +99,11 @@ function set_up() {
     mkdir -p .rally_it/cache
     cd .rally_it/cache
     if [ ! -f elasticsearch-"${ES_METRICS_STORE_VERSION}".tar.gz ]; then
-        curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-"${ES_METRICS_STORE_VERSION}".tar.gz
+        # If curl fails immediately, executing all retries will take up to (2**retries)-1 seconds.
+        curl --retry 8 -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-"${ES_METRICS_STORE_VERSION}".tar.gz || { rm elasticsearch-"${ES_METRICS_STORE_VERSION}".tar.gz; exit 1; }
```

Comment:
Maybe it makes sense to extract the artefact name (`elasticsearch-"${ES_METRICS_STORE_VERSION}".tar.gz`) as a constant as it is repeated multiple times now?

Reply:
Makes sense, will address.

Reply:
I pushed 20972a29aa086e4d5b1b544c8b74bb5c5ecb5011 for ^^